packages/wrangler/src/type-generation.ts

```diff
@@ -112,6 +112,54 @@ export async function typesHandler(
 	);
 }
 
+/**
+ * Check if a string is a valid TypeScript identifier. This is a naive check and doesn't cover all cases
+ * If not, it should be wrapped in quotes
+ */
+export function isValidIdentifier(key: string) {
+	return /^[a-zA-Z_$][a-zA-Z0-9_$]*$/.test(key);
```

Comment:
`\w` is equivalent to `[a-zA-Z0-9_$]` no?
```suggestion
	return /^[a-zA-Z_$]\w*$/.test(key);
```

Reply:
`\w` doesn't include `$`, but I could do `\w$`, yep!

Reply:
Done!

Reply:
ah yeah sorry I missed the `$` 😅 (which I included in the comment as I just copy-pasted the thing 🙈)